jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -48,7 +48,7 @@ do
 done
 
 mkdir -p "${LOGS_DIR}/qemu"
-sudo cp -r /var/log/libvirt/qemu/* "${LOGS_DIR}/qemu/"
+sudo sh -c "cp -r /var/log/libvirt/qemu/* "${LOGS_DIR}/qemu/""
```

Comment:
There is an issue with the quotes here. you should escape the inner ones, otherwise ${LOGS_DIR/qemu/ is not quoted and you have a useless double quote marks at the end. @Jaakko-Os Can you please fix this in a new PR ?

Reply:
https://github.com/metal3-io/project-infra/pull/140